src/cc/reno.rs

```diff
@@ -95,6 +103,29 @@ impl cc::CongestionControl for Reno {
     ) {
         self.bytes_in_flight -= packet.size;
 
+        if let Some(congestion_recovery_start_time) =
+            self.congestion_recovery_start_time
+        {
+            // End of current recovery episode.
+            self.congestion_recovery_start_time = None;
+
+            // Simple lost detection: when the recovery episode ends
+            // and the duration of current recovery episode is
+            // short than a given threshold, currently defined by
+            // 2 x min_rtt. This means there was a small loss
+            // recovered very quickly, indicating this has a
+            // low probability of congestion but a simple packet lost.
+            // In this case we restore previous cwnd/ssthresh
+            // to resume faster after recovery.
+            if packet.time > congestion_recovery_start_time &&
+                congestion_recovery_start_time.elapsed() <
+                    SIMPLE_LOST_THRESHOLD * _min_rtt
```

Comment:
Realted to my comment on the other PR https://github.com/cloudflare/quiche/pull/299#discussion_r359852031

if using `_min_rtt` you can change it to `min_rtt`,

Reply:
Fixed. Thanks!